edu.cuny.citytech.refactoring.common.eval/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eval
```

Comment:
Should be externalized ala https://github.com/ponder-lab/Common-Eclipse-Java-Refactoring-Framework/blob/50907b136723a78d0fd10a6c1ba1fe2bc0f3aefa/edu.cuny.citytech.refactoring.common.core/META-INF/MANIFEST.MF#L7.

Reply:
Not too sure what this means but will look into it

Reply:
https://github.com/ponder-lab/Common-Eclipse-Java-Refactoring-Framework/blob/50907b136723a78d0fd10a6c1ba1fe2bc0f3aefa/edu.cuny.citytech.refactoring.common.core/OSGI-INF/l10n/bundle.properties#L3.

Reply:
You can change it to Hunter if you like.